.github/workflows/gradle-check.yml

```diff
@@ -48,6 +51,8 @@ jobs:
           echo "pr_to_clone_url=$repo_url" >> $GITHUB_ENV
           echo "pr_title=Push trigger $branch_name $ref_id $repo_url" >> $GITHUB_ENV
           echo "pr_number=Null" >> $GITHUB_ENV
+          echo "pr_owner=Null" >> $GITHUB_ENV
```

Comment:
Nit: Can we omit values that are not set?

Reply:
I believe this `Null` was added to easily handle the variable in jenkins pipeline script. 
We can leave it as is now, since changing this will also result in changes in the `gradle-check` jenkins pipeline script.